apps/expo/app/(drawer)/_layout.tsx

```diff
@@ -1,9 +1,14 @@
-import { useSession } from 'app/context/Auth/SessionProvider';
-import { useProtectedRoute } from 'app/hooks/auth/useProtectedRoute';
-import { Navigation } from 'app/components/navigation';
+import { Redirect, Stack } from 'expo-router';
+import { AuthLoader } from 'app/auth/AuthLoader';
+import { Text } from '@packrat/ui';
 
 export default function AppLayout() {
-  const { session } = useSession();
-  useProtectedRoute(session);
-  return <Navigation />;
+  return (
+    <AuthLoader
```

Comment:
Not sure why but this breaks native for some reason, despite logic being quite similar to what we had before